spec/models/placements/academic_year_spec.rb

```diff
@@ -0,0 +1,60 @@
+# == Schema Information
+#
+# Table name: academic_years
+#
+#  id         :uuid             not null, primary key
+#  ends_on    :date
+#  name       :string
+#  starts_on  :date
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+require "rails_helper"
+
+RSpec.describe Placements::AcademicYear, type: :model do
+  describe ".current" do
+    let!(:current_academic_year) { create(:placements_academic_year, :current) }
+
+    it "returns the academic year for the current date" do
+      expect(described_class.current).to eq(current_academic_year)
+    end
+  end
+
+  describe "#next" do
+    let!(:current_academic_year) do
+      create(:placements_academic_year,
+             starts_on: Date.parse("1 September 2024"),
+             ends_on: Date.parse("31 August 2025"),
+             name: "2024 to 2025")
+    end
+    let!(:next_academic_year) do
+      create(:placements_academic_year,
+             starts_on: Date.parse("1 September 2025"),
+             ends_on: Date.parse("31 August 2026"),
+             name: "2025 to 2026")
+    end
+
+    it "returns the next academic year" do
+      expect(current_academic_year.next).to eq(next_academic_year)
+    end
+  end
+
+  describe "#previous" do
+    let!(:current_academic_year) do
+      create(:placements_academic_year,
+             starts_on: Date.parse("1 September 2024"),
+             ends_on: Date.parse("31 August 2025"),
+             name: "2024 to 2025")
+    end
+    let!(:previous_academic_year) do
+      create(:placements_academic_year,
+             starts_on: Date.parse("1 September 2023"),
+             ends_on: Date.parse("31 August 2024"),
+             name: "2023 to 2024")
+    end
+
+    it "returns the previous academic year" do
+      expect(current_academic_year.previous).to eq(previous_academic_year)
+    end
+  end
```

Comment:
Could you also add tests for it the previous/next academic year does not exist, then the academic year is created.